### ISSUE TITLE
Improve Code of Conduct

### ISSUE THREAD

**Issue by AlvYuste** [AlicanteFrontend/alicantefrontend.github.io#25]
We try to make the CoC more concise and effective, taking as examples some other mature communities:

- [Twitter](https://github.com/twitter/code-of-conduct/blob/master/code-of-conduct.md)
- [Python Spain](https://github.com/python-spain/web/blob/master/CODE_OF_CONDUCT.md)
- [Codenares](https://github.com/codenares/codenares/wiki/C%C3%B3digo-de-conducta)
- [Google Events](https://www.google.com/events/policy/anti-harassmentpolicy.html)
- [Ubuntu](https://www.ubuntu.com/about/about-ubuntu/conduct)

**Comment by keropodium** [comment]
The first step before enhancing the document should be to decide whether we want to split it onto a organisational one, having one, editing this one by maintaining both or only this one.

**Comment by kutyel** [comment]
There is a big gap in this CoC that is not covered but it is addressed in the [Codenares](https://github.com/codenares/codenares/wiki/C%C3%B3digo-de-conducta), referring to the **technical level** as source of discrimination. As a technical community, we sure want to avoid this kind of harassment! 😉 

On a separate thread, I think we can simplify the whole document and make it look closer to the Google one, leaving most of the explicit things to the criteria of well being and conscience of the average person. Keep up the good work! 👍

**Comment by AlvYuste** [comment]
@keropodium Good point. Placing it at org level has more sense to me. We should create a dedicated repo, right? like this one: https://github.com/twitter/code-of-conduct. When the decision is taken we may continue the discussion in that new repository.

@kutyel You are right, including the technical level tolerance would be a good contribution. And making the whole document more concise is definitely a goal, we have to focus on that.

Thanks for the comments! 😄

**Comment by aarongarciah** [comment]
@kutyel totally agree about the technical level issue.
